src/lib/remark-plugins/rehype-sanitize/__tests__/rehype-sanitize.test.ts

```diff
@@ -5,10 +5,13 @@
 
 import fs from 'fs'
 import path from 'path'
-import type { Root, RootContent, Node } from 'hast'
```

Comment:
🚧 Had to update this to not really be Typescript to get tests to work in the only way I could find. There's probably a better way.

Reply:
✅ Updated, now back to TypeScript thanks to Kevin's pointer on the dynamic jest config in `web-platform-packages` 🙌